scripts/tb-bandwidth.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+#
+# dp_tunnel_status() - Reads DisplayPort tunnel status
+# @domain: Domain number
+# @router: Route string of the router
+# @adapter: DisplayPort IN adapter
+#
+dp_tunnel_status() {
```

Comment:
Can you re-use this with the dp-tunnel-status.sh?

Reply:
I have made new changes and uploaded changes. please review new changes